checkov/terraform/checks/resource/gcp/GoogleCloudSqlBackupConfiguration.py

```diff
@@ -1,17 +1,28 @@
-from checkov.common.models.enums import CheckCategories
-from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceCheck
 
 
-class GoogleCloudSqlBackupConfiguration(BaseResourceValueCheck):
+class GoogleCloudSqlBackupConfiguration(BaseResourceCheck):
     def __init__(self):
         name = "Ensure all Cloud SQL database instance have backup configuration enabled"
         id = "CKV_GCP_14"
         supported_resources = ['google_sql_database_instance']
         categories = [CheckCategories.NETWORKING]
```

Comment:
```suggestion
        categories = [CheckCategories.BACKUP_AND_RECOVERY]
```
could you change the category, this fits better 🙂

Reply:
👍